### ISSUE TITLE
Function to post-process result file with new strictness/selection criteria

### ISSUE THREAD

**Issue by emiliesg** [pharmpy/pharmpy#3201]
### Feature request

Suggest to add a function to apply other strictness criteria to the results of a subtool and/or change selection criteria (e.g. BIC vs mBIC, E value for mBIC, etc.) and redo the ranking of the models according to these criteria

**Comment by rikardn** [comment]
I agree that this is a good idea. We already have an idea on how this could be done, but it needs to be fleshed out a bit more before we can start implementing.

**Comment by stellabelin** [comment]
A new tool `ModelRank` has been implemented that should allow for this (if you mean to after a full tool run explore how it would have behaved with different selection criteria or strictness). Tool and relevant user guide will be in the next release!
